metadata-models/src/main/resources/entity-registry.yml

```diff
@@ -693,6 +693,13 @@ plugins:
       supportedEntityAspectNames:
         - entityName: '*'
           aspectName: '*'
+    - className: 'com.linkedin.metadata.aspect.validation.UserDeleteValidator'
```

Comment:
This method is deprecated use the the `SpringStandardPluginConfiguration.java` to define the plugin